src/parsers/route.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2020-present Daniel Trugman
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include "pfs/parsers.hpp"
+#include "pfs/utils.hpp"
+#include "pfs/types.hpp"
+
+#include <set>
+#include <sstream>
+
+namespace pfs {
+namespace impl {
+namespace parsers {
+
+route parse_route_line(const std::string& line)
```

Comment:
Since the file is located under `/proc/net`, the file should be called `net_route.cpp` and the method `parse_net_route_line`